nvme-print-stdout.c

```diff
@@ -4483,6 +4490,10 @@ static void stdout_feature_show_fields(enum nvme_features_id fid,
 			       host_behavior->etdas ? "True" : "False");
 			printf("\tLBA Format Extension Enable (LBAFEE): %s\n",
 			       host_behavior->lbafee ? "True" : "False");
+			printf("\tCopy Descriptor Format 2h Enabled: %s\n",
+			       (buf[4] & (1 << 2)) ? "True" : "False");
```

Comment:
Can you use the field definition you added in libnvme instead of accessing the raw byte index?

Reply:
```
$ nvme get-feature --feature-id 0x16 /dev/nvme0 --human-readable
get-feature:0x16 (Host Behavior), Current value:00000000
        Advanced Command Retry Enable (ACRE): False
        Extended Telemetry Data Area 4 Supported (ETDAS): False
        LBA Format Extension Enable (LBAFEE): False
        Copy Descriptor Format 2h Enabled (CDFE): False
        Copy Descriptor Format 3h Enabled (CDFE): False
```